### PR TITLE
By default, do not create a tempstorage on Plone 6.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
See https://github.com/plone/plone.recipe.zope2instance/issues/180, and lots more info in https://github.com/plone/Products.CMFPlone/issues/2957.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
I often forget: there is no need to run the Jenkins tests, because this recipe is not tested there.

**Comment by mauritsvanrees** [comment]
Can someone review this please?
Note again: all GHA tests pass, but I did not start Jenkins tests because this package is not actually tested on Jenkins.

File: src/plone/recipe/zope2instance/recipe.py [hunk 1]
```diff
@@ -698,7 +699,17 @@ def is_rs_option(name):
         if options.get("storage-wrapper"):
             storage_snippet = indent(options["storage-wrapper"] % storage_snippet, 4)
 
-        zodb_tmp_storage = options.get("zodb-temporary-storage", "on")
+        zodb_tmp_storage = options.get("zodb-temporary-storage")
+        if zodb_tmp_storage is None:
+            # What should be the default?
+            # In Plone 6 we do not want to use the temporary storage anymore.
+            # See https://github.com/plone/plone.recipe.zope2instance/issues/180
+            requirements, ws = self.egg.working_set(["plone.recipe.zope2instance"])
```
Comment (davisagli) [inline src/plone/recipe/zope2instance/recipe.py:707]: Calculating the working set is presumably expensive. Is it cached in some way between here and `install_scripts`?

**Review by davisagli — COMMENTED** [review]
Mostly LGTM; one question about performance impact

File: src/plone/recipe/zope2instance/recipe.py (hunk 1 above)
Comment (mauritsvanrees) [reply src/plone/recipe/zope2instance/recipe.py:707]: Good question.

I timed it with `time.time()`. Getting the working set here takes 3.0 seconds. And then when installing the scripts it takes 0.007 seconds. So setuptools/buildout/whoever is doing the caching for us. :-)

**Review by davisagli — APPROVED** [review]

**Comment by mauritsvanrees** [comment]
I have merged it. Thanks for the review David!

**Pull request merged**